code.gs

```diff
@@ -1874,7 +1874,7 @@ function generateEmailNotification (forceDate) {
         ) ||
         (
           eventData['goo.contactsProfileId'] !== null &&
-          eventData['goo.contactsProfileId'] === contactList[i]
+          eventData['goo.contactsProfileId'] === contactList[i].gPlusId
```

Comment:
Appending the property here doesn't immediately seem to have anything to do with the SELF birthdays issue. If it does in some obscure way I recommend add a line to the git-commit explaining it, because it has had me scratching my head and grepping for a few minutes, to see if it was just a mistake.

Reply:
Yes, it was related. I've now added a brief explaination in the commit.